src/request_tracing.cpp

```diff
@@ -191,6 +193,16 @@ RequestTracing::RequestTracing(ngx_http_request_t *request,
                  "starting Datadog request span for %p", request_);
 
   dd::SpanConfig config;
+  if (srv_conf_->service_name) {
```

Comment:
Do we have the capability to do trace stats calculations in the tracer? If we don't, then there are no potential issues. However, if we do then we will have to be careful about how we calculate if a span is "top-level" or not, now that we have the global tracer configuration set to service name `nginx` and the individual spans may receive a custom service name

Reply:
➕

Reply:
Care to explain?